SpbuCsharpHw/hw1/Sort.cs

```diff
@@ -2,6 +2,16 @@
 {
     public class Sort
     {
-        
+        public static void BubbleSort(int[] array)
+        {
+            for (var i = 0; i < array.Length; i++)
+            {
+                for (var j = i + 1; j < array.Length; j++)
+                {
+                    if (array[i] > array[j])
+                        (array[i], array[j]) = (array[j], array[i]);
```

Comment:
```suggestion
                    if (array[i] > array[j])
                    {
                        (array[i], array[j]) = (array[j], array[i]);
                    }
```
Скобки лучше ставить даже если внутри if (for и т.д.) всего один оператор, спасает от ошибок.